### DISCUSSION TITLE
S2默认打开的排序功能升级到2.0后样式发生了变化，而且有时候位置对不齐？

### DISCUSSION THREAD

**Discussion by Zhaohanqi12** [antvis/S2#2730] [General]
![image](https://github.com/antvis/S2/assets/166707652/ccd24b36-1327-4392-a518-9ec224dce165)
2.0版本多了几个三角标志，
![image](https://github.com/antvis/S2/assets/166707652/9d3d29ee-f9ab-485c-979a-a1d270b81e07)
不对齐案例
![image](https://github.com/antvis/S2/assets/166707652/937d8249-3f90-4d47-a398-5b896911ba02)
怎么可以把三角标识去掉？

**Comment by lijinke666** [comment]
这个排序菜单就是 DOM实现的, 自行用 CSS 覆盖隐藏箭头即可.

**Reply by Zhaohanqi12** [reply]
好的，谢谢
